.github/workflows/ci.yml

```diff
@@ -91,7 +91,7 @@ jobs:
           path: deploy/
 
   deploy:
-    if: github.ref == 'refs/heads/main'
+    if: github.event_name != 'pull_request' && github.ref == 'refs/heads/master'
```

Comment:
?

Reply:
`master` -> `main` is now fixed, but PRs will run on `refs/heads/main` so we need to ignore PRs to avoid deploying on them.